test/mainnet-fork/PsmCalls.t.sol

```diff
@@ -67,6 +67,43 @@ contract MainnetControllerSwapUSDSToUSDCTests is ForkTestBase {
         assertEq(dai.allowance(address(almProxy),  address(PSM)),     0);
     }
 
+    function test_swapUSDSToUSDC_rateLimited() external {
+        bytes32 key = mainnetController.LIMIT_USDS_TO_USDC();
+        vm.startPrank(relayer);
+
+        // Rate limited so split this in two
```

Comment:
I'd rather make minting unlimited for this test

Reply:
Not sure I understand.

Reply:
Sure